src/prefect/cli/auth.py

```diff
@@ -0,0 +1,68 @@
+import os
+
+import click
+import toml
+
+from prefect import Client
+
+
+@click.group(hidden=True)
+def auth():
+    """
+    Handle Prefect Cloud authorization.
+
+    \b
+    Usage:
+        $ prefect auth [COMMAND]
+
+    \b
+    Arguments:
+        add     Add a Cloud auth token to your Prefect config
+
+    \b
+    Examples:
+        $ prefect auth add --token MY_TOKEN
+
+    \b
+        $ prefect auth add --token MY_TOKEN --config-path ~/.prefect/new-config.toml
+    """
+    pass
+
+
+@auth.command(hidden=True)
+@click.option(
+    "--token", "-t", required=True, help="A Prefect Cloud auth token.", hidden=True
+)
+@click.option(
+    "--config-path",
+    "-c",
+    default="~/.prefect/config.toml",
```

Comment:
You should use `os.expanduser` on this path

Reply:
Good idea

Reply:
Oh I just realized I already do use `expanduser` on the first line of this function

Reply:
ah fair